test/index.js

```diff
@@ -67,4 +68,20 @@ describe('Sitemap generator', () => {
       result.should.be.ok;
     });
   });
+
+  it('IDN handling', () => {
+    hexo.config.url = 'http://fôo.com/bár';
```

Comment:
While this option can be applied, but the following `result` still show "http://yoursite.com/".

Reply:
I think it's related to `describe()` `it()`. But I don't know how to proceed.

Reply:
could also related to how [urlObject](https://nodejs.org/api/url.html#url_legacy_url_api) is referenced.